### PR TITLE
Skip socks proxy tests.

### PR DESCRIPTION
**Pull request by RatanShreshtha**
Skipping the tests for socks proxies now as of https://github.com/python-trio/urllib3/issues/77.

### CODE REVIEW

**Comment by pquentin** [comment]
Thanks!

Can you please:

 * [skip at the module level using pytestmark](https://docs.pytest.org/en/latest/skipping.html#skip-all-test-functions-of-a-class-or-module)
 * omit `*urllib3/contrib/socks.py` in `.coveragerc`
 * and remove the xfail annotations?

I think we also need to mention in the README that we don't support SOCKS, pyOpenSSL and SecureTransport.

**Comment by RatanShreshtha** [comment]
I have skipped the socks proxy test at module level and omitted omit `*urllib3/contrib/socks.py` in `.coveragerc`.

File: test/contrib/test_socks.py [hunk 1]
```diff
@@ -693,8 +683,7 @@ class TestSOCKSWithTLS(IPV4SocketDummyServerTestCase):
     """
     Test that TLS behaves properly for SOCKS proxies.
     """
-    @pytest.mark.skip
-    @pytest.mark.skipif(not HAS_SSL, reason='No TLS available')
```
Comment (pquentin) [inline test/contrib/test_socks.py]: I believe this one comes from the master branch and should stay.

**Comment by pquentin** [comment]
Can you also add the mention in the README? Thanks.

**Comment by RatanShreshtha** [comment]
yeah sure

**Comment by RatanShreshtha** [comment]
thanks

**Pull request merged**